### ISSUE TITLE
Nice!

### ISSUE THREAD

**Issue by ghost** [Ingan121/BasicThemer2#28]
nice thing i can make my theme look like windows 7 basic!
can you do aerothemer?

**Comment by kitty93341-commits** [comment]
No, the aero theme is not in the files of windows but the transparency is
